main.go

```diff
@@ -335,7 +339,7 @@ func main() {
 		return
 	}
 
-	km, err := NewKeyManager(128, *pass)
+	km, err := NewKeyManager(128, *pass, *mnemonic)
```

Comment:
hi @daroay, thanks for your contribution!
but we should change the bitSize depends on mnemonic phrases here because of: 

// bitSize has to be a multiple 32 and be within the inclusive range of {128, 256}
// 128: 12 phrases
// 256: 24 phrases

Reply:
so 
12 -> 128
13 -> 160
14 -> 192
...
256 -> 256

Please let me know if this is the case, I can add a simple function to calculate this.

Reply:
oh sorry, my mistake.
if you put a existing mnemonic, then will not use bitSize at all.